tests/components/wallbox/test_sensor.py

```diff
@@ -27,3 +31,11 @@ async def test_wallbox_sensor_class(hass):
     assert state.state == "25.0"
 
     await hass.config_entries.async_unload(entry.entry_id)
+
+
+async def test_wallbox_sensor_class_rounding_error(hass):
+    """Test wallbox sensor class rounding error."""
+
+    await setup_integration_rounding_error(hass)
+
+    await hass.config_entries.async_unload(entry.entry_id)
```

Comment:
Where is the error tested?

Reply:
sorry for the late response, guess I missed this. The setup_integration_rounding_error return faulty data from the mock api which in turn triggers an exception that is being handled in the code.